upsidedown/parts/footer.html

```diff
@@ -0,0 +1,41 @@
+<p>Blogs tend to have a very common structure. Upsidedown is not here to repeat old formulas as we, the humans, do not fit into the same boxes. This Theme Design is not here for certainty but freshness. It is here to invert things, proposing a brand new way to convey personal blogs. More than being yet another WordPress blog theme, Upsidedown is designed in Full Site Editor and it is full of ambition. Maybe it is the best choice for your blog.</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"verticalAlignment":"top","width":"25%","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"blockGap":"0px"}}} -->
+<div class="wp-block-column is-vertically-aligned-top" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px;flex-basis:25%"><!-- wp:paragraph -->
+<p><a rel="noreferrer noopener" href="mailto:name@provider.com" target="_blank"><span style="text-decoration: underline;">Email the author</span></a> and share this blog theme with your colleagues. Enjoy!</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:spacer {"height":"2rem","style":{"spacing":{"margin":{"top":"0px","bottom":"0px"}}}} -->
+<div style="margin-top:0px;margin-bottom:0px;height:2rem" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:columns {"style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0px","bottom":"0px"},"blockGap":{"top":"30px","left":"30px"}}}} -->
+<div class="wp-block-columns" style="margin-top:0px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:column {"width":"75%","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"blockGap":"0px"}}} -->
+<div class="wp-block-column" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px;flex-basis:75%"><!-- wp:site-title {"style":{"typography":{"letterSpacing":"0px","lineHeight":1.4,"fontStyle":"normal","fontWeight":"700","textTransform":"uppercase"}},"fontSize":"small"} /--></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"verticalAlignment":"center","width":"25%","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"blockGap":"0px"}}} -->
+<div class="wp-block-column is-vertically-aligned-center" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px;flex-basis:25%"><!-- wp:paragraph -->
+<p>© Designed with <a href="https://wordpress.org/" target="_blank" rel="noreferrer noopener">WordPress</a></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:spacer {"height":"4.8em","style":{"spacing":{"margin":{"top":"0px","bottom":"0px"}}}} -->
+<div style="margin-top:0px;margin-bottom:0px;height:4.8em" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
```

Comment:
The contents of this file should be in a pattern, so we can make the strings translatable.